authorship/data.py

```diff
@@ -151,42 +152,45 @@ def get_frequent_words(speakers, n):
     return freq[:n]
 
 
-def filter_texts_size(speakerdict, wordlist, tokens_per_sample):
+def filter_texts_size(speakerdict, wordlist):
     """
-    it returns per speaker all possible samples (the length of the samples is equal to the lenght of the wordlist and
-    its values are the frequency of the words in the wordlist for that speaker/sample).
+    it returns one conversation of one speaker  (the length of the samples is equal to the length of the wordlist and
+    its values are the frequency of the words in the wordlist for that speaker/conversation).
 
     :param speakerdict: dict of all words used per speaker
     :param wordlist: the n most freq words in corpus
-    :param tokens_per_sample: number of words within a sample
     """
     filters = [
         ExtractWords(),
```

Comment:
`ExtractWords` doesn't seem to do anything. `RearrangeSamples` doesn't rearrange anything.

Reply:
both removed